src/StackOverflowSolutionProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Astrotomic\IgnitionStackOverflowTab;
+
+use Throwable;
+use Illuminate\Support\Str;
+use Facade\IgnitionContracts\BaseSolution;
+use Facade\IgnitionContracts\HasSolutionsForThrowable;
+
+class StackOverflowSolutionProvider implements HasSolutionsForThrowable
+{
+    public function canSolve(Throwable $throwable): bool
+    {
+        return true;
+    }
+
+    public function getSolutions(Throwable $throwable): array
+    {
+        $url = curl_init('https://api.stackexchange.com/2.2/search/advanced?&pagesize=5&order=desc&sort=relevance&site=stackoverflow&accepted=True&page=1&filter=!9YdnSJ*_T&q='.urlencode($throwable->getMessage()));
```

Comment:
This could be made somewhat nicer in my opinion. Maybe like this:
```
$query = http_build_query([
    'page' => 1,
    'pagesize' => 5,
    'order' => 'desc',
    'sort' => 'relevance',
    'site' => 'stackoverflow',
    'accepted' => 'True',
    'filter' => '!9YdnSJ*_T',
    'q' => urlencode($throwable->getMessage()),
]);
$url = 'https://api.stackexchange.com/2.2/search/advanced?' . $query;
$curl = curl_init($url);
```
Making the URL and the static parameters constants of the class would probably be even better.

Using `$url` as name for a cURL instance seems weird.

A note about `curl_*` in general: currently, there is no requirement for `ext-curl` in `composer.json`. This should be added. Alternatively, guzzle could be used for the request (it would be a little nicer).

Reply:
Hey, just before it gets "fixed": no dependency for a simple curl request. The other stuff is fine. 
But it's still an error page - every additional dependency/code should be "unbreakable".

Reply:
I'm sorry to disagree, but this is still an optional package to the (soon-to-be) standard error page. Additional dependencies are not an issue and should be required accordingly. If suggestions are a dev-only thing, it's also fine to require ext-curl as dev-dependency.

`guzzlehttp/guzzle` would be a dev-dependency of the framework already. Using guzzle would hence make sense. It should still be required in this package as well.

Reply:
Sorry, adding `ext-curl` is required. But I don't want guzzle for it. I love guzzle but it's too much for this simple task.

Reply:
Alright, then I got it wrong. Thanks for clearifying.

Reply:
No problem. Be assured that I will do my best to prevent any bug in PHP code.
I know about the responsibility I have with this package. Why I am/will be this hard with the PHP part of the package:
All the code is triggered after something bad happened in the app. So every code that creates an error afterwards will force a blank screen of death. And this would be the exact opposit of what ignition and this package should do.